src/frontend/web_application/src/components/SignupForm/presenter.jsx

```diff
@@ -186,6 +232,31 @@ class SignupForm extends Component {
             </FormColumn>
           </FormRow>
           <FormRow>
+            <FormColumn bottomSpace>
+              <div className="s-signup__privacy">
+                <h4>{__('signup.form.privacy.title')}</h4>
+                <p className="s-signup__privacy__text">
+                  {__('signup.form.privacy.intro')}
+                  <PrivacyLink
```

Comment:
no need to create a new component I think (and I don't like much create a new component at runtime, I mean directly in `render`)